examples/quickstart/00-rbac.json

```diff
@@ -0,0 +1,70 @@
+{
+   "apiVersion": "v1",
+   "items": [
+      {
+         "apiVersion": "v1",
+         "kind": "Namespace",
+         "metadata": {
+            "name": "heptio-sonobuoy"
+         }
+      },
+      {
+         "apiVersion": "v1",
+         "kind": "ServiceAccount",
+         "metadata": {
+            "labels": {
+               "component": "sonobuoy"
+            },
+            "name": "sonobuoy-serviceaccount",
+            "namespace": "heptio-sonobuoy"
+         }
+      },
+      {
+         "apiVersion": "rbac.authorization.k8s.io/v1beta1",
```

Comment:
Can we somehow option wrap rbac with a default of true but allow for disabled generation per other issue filed?